mne/tests/test_epochs.py

```diff
@@ -2450,6 +2450,48 @@ def test_events_list():
     assert_array_equal(epochs.events, np.array(events))
 
 
+def test_save_overwrite(tmpdir):
+    """Test saving with overwrite functionality."""
+
+    tempdir = str(tmpdir)
+    raw = mne.io.RawArray(np.random.RandomState(0).randn(100, 10000),
+                          mne.create_info(100, 1000.))
+
+    events = mne.make_fixed_length_events(raw, 1)
+    epochs = mne.Epochs(raw, events)
+
+    # scenario 1: overwrite=False (default) and there isn't a file to overwrite
+    # make a filename that has not already been saved to
+    fname1 = op.join(tempdir, 'test' + str(np.random.rand()) + '-epo.fif')
```

Comment:
I don't think it's necessary to use random numbers here. Just make explicit names.

'test_v1-epo.fif'
'test_v2-epo.fif'

etc.

Reply:
Thanks! I made that edit.